custom/covid/management/commands/update_owner_ids.py

```diff
@@ -24,7 +24,7 @@ def case_block(self, case, child_location):
             owner_id=child_location.location_id,
         ).as_xml()).decode('utf-8')
 
-    def update_cases(self, domain, case_type, user_id):
+    def update_cases(self, domain, case_type, user_id, active_location):
```

Comment:
it would be great if these commands that did not use this argument did not pass it into the method or have to parse it. One way to do that is saving location on `self` in the command above, so that it doesn't need to be an extra argument.